src/corner-dialog/src/CornerDialog.js

```diff
@@ -134,7 +134,7 @@ export default class CornerDialog extends PureComponent {
     /**
      * Props that will set position of corner dialog
      */
-    position: PropTypes.oneOf(AbsolutePosition)
+    position: PropTypes.string
```

Comment:
This should be more like so: https://github.com/segmentio/evergreen/blob/1e2051e4ab71ae3f13331d8a841ac85bcaf6a2e6/src/popover/src/Popover.js#L13-L22

Except we only want the corners.